openpype/modules/kitsu/utils/sync_service.py

```diff
@@ -149,6 +177,21 @@ def _new_asset(self, data):
         # Update
         self._update_asset(data)
 
+        # Print message
+        ep = None
+        ep_id = asset.get("episode_id")
+        if ep_id and ep_id != "":
+            ep = gazu.asset.get_episode(ep_id)
```

Comment:
Using a `else` statement is faster and more explicit about what is going on.

```suggestion
        ep_id = asset.get("episode_id")
        if ep_id and ep_id != "":
            ep = gazu.asset.get_episode(ep_id)
         else:
             ep = None
```